exercises/exercise3.js

```diff
@@ -1,12 +1,126 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Say Hello to Babushka!',
+    prompt: "Hello, Babushka",
     handler: function () {
-      return 'Good Evening to You!';
+      return 'Privyet! What Babushka can do for you?';
     },
+    isReply: true,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+
+  {
+    label: 'censorship',
+    prompt: "#{name1}",
+    handler: function (HRparams) {
+      if (HRparams.name1 === "Fuck" || "bitch"); {
+        return  "That's no way of approaching an old Babushka, try again."
+      }
+    },
+    isReply: true,
+    isCaseSensitive: false,
+    isListening: true,
+   }, 
+
+   {
+    label: 'Subject',
+    prompt: "Babushka, I need help with #{name1}",
+    handler: function (HRparams) {
+      return "You might not deserve some" + HRparams.name1 + "but Babushka will help you anyway."
+    },
+    isReply: true,
+    isCaseSensitive: false,
+    isListening: true,
+   }, 
+
+   {
+    label: 'Reibursement',
+    prompt: 'Babushka, please expense the amount of $#{amount1 as number} to #{name1} ',
+    handler: function (params) {
+      if (params.amount1 > 75) {
+        return "Please, send a PDF of the receipt to hr@keepingcool.com"
+      }else if (params.amount1 < 75)
+      return "Thank you" + params.name1 + "," + " " + "Babushka will take care of that for you."
+
+    },
+    isReply: true,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+
+  {
+    label: 'Email addresses',
+    prompt: '#{name1} ',
+    handler: function (params) {
+      if (params.name1 === "Ken" || "Connie") {
+        return "Before botheting the CEOs, consider bothering someone else";
+        } else{
+    
+          switch (params) {
```

Comment:
On line 63 you're only passing params into the switch when you should be passing the 'params.name1'